### PR TITLE
ARUHA-1457 Initialize subscription on patch request.

### PR DESCRIPTION
**Pull request by antban**
ARUHA-1457 Initialize subscription on patch request.

## Description
In case, if client wants to PATCH subscription before streaming (initialization of cursors) nakadi will initialize cursors beforehands

## Deployment Notes
No additional stuff to do

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/subscription/state/StartingState.java [hunk 1]
```diff
@@ -56,13 +57,9 @@ public void onEnter() {
      */
     private void createSubscriptionLocked() {
         // check that subscription initialized in zk.
```
Comment (lmontrieux) [inline src/main/java/org/zalando/nakadi/service/subscription/state/StartingState.java]: I think this isn't correct anymore?

**Comment by antban** [comment]
@lmontrieux Done https://github.com/zalando/nakadi/pull/823/commits/7964db40e8592e47415c9eaef2ae975bcf45e04a

**Comment by lmontrieux** [comment]
👍

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
deploy validation please

**Comment by antban** [comment]
:+1:

**Comment by lmontrieux** [comment]
👍

**Pull request merged**